diff --git a/gui/src-tauri/src/main.rs b/gui/src-tauri/src/main.rs
index b64e6974..0d5631f4 100644
--- a/gui/src-tauri/src/main.rs
+++ b/gui/src-tauri/src/main.rs
@@ -274,6 +274,8 @@ fn setup_tauri(
 
 			app.manage(Mutex::new(window_state));
 
+			return Ok(());
+
 			if cli.skip_server_start_if_running && server_running() {
 				log::info!("Skipping server start: server is already running.");
 				return Ok(());
