diff --git a/src/plugins/_core/supportHelper.tsx b/src/plugins/_core/supportHelper.tsx
index 825bbc20..f598b501 100644
--- a/src/plugins/_core/supportHelper.tsx
+++ b/src/plugins/_core/supportHelper.tsx
@@ -137,7 +137,7 @@ const settings = definePluginSettings({}).withPrivateSettings<{
 
 export default definePlugin({
     name: "SupportHelper",
-    required: true,
+    required: false,
     description: "Helps us provide support to you",
     authors: [Devs.Ven],
     dependencies: ["UserSettingsAPI"],
